### ISSUE TITLE
Support serverless-webpack 2.1.0

### ISSUE THREAD

**Issue by joncursi** [ajmath/serverless-offline-scheduler#8]
After upgrading to [`serverless-webpack@2.1.0`](https://github.com/elastic-coders/serverless-webpack/releases/tag/v2.1.0), all of my scheduled jobs stopped running with the following error:

```
Serverless: scheduler: unable to find source for <JOB_NAME>
```

**Comment by ajmath** [comment]
Do they ever start up successfully?  The next time the schedule kicks in the job should start up.  Maybe there are some better triggers that can be used.

**Comment by joncursi** [comment]
I'm not sure what you mean but I'm having the same issue on `serverless-webpack@2.2.0`. This is the full output I get in console:

```
Serverless: Bundling with Webpack...
Time: 940ms
     Asset     Size  Chunks             Chunk Names
handler.js  70.9 kB       0  [emitted]  main
   [0] external "babel-runtime/regenerator" 42 bytes {0} [not cacheable]
   [2] external "redbird-core/lib/constants" 42 bytes {0} [not cacheable]
   [4] ./src/helpers/connectToDatabase.js 1.57 kB {0} [built]
   [5] external "async/queue" 42 bytes {0} [not cacheable]
   [6] external "babel-runtime/core-js/json/stringify" 42 bytes {0} [not cacheable]
   [7] external "aws-sdk" 42 bytes {0} [not cacheable]
   [8] ./src/lambdas/integrationJobWorker/index.js 7.18 kB {0} [built]
  [12] ./src/lambdas/storyFanOutJobWorker/index.js 6.96 kB {0} [built]
  [13] ./src/handler.js 1.17 kB {0} [built]
  [14] ./src/lambdas/integrationJobCreator/index.js 3.92 kB {0} [built]
  [33] ./src/lambdas/integrationJobWorker/updateProfileValue.js 2.83 kB {0} [built]
  [36] ./src/lambdas/integrationJobWorker/updateTickerValue.js 2.47 kB {0} [built]
  [37] ./src/lambdas/storyFanOutJobCreator/index.js 3.92 kB {0} [built]
  [38] ./src/lambdas/storyFanOutJobWorker/addToNewsFeed.js 3.05 kB {0} [built]
  [39] ./src/lambdas/storyFanOutJobWorker/updateFanOutStatus.js 2.45 kB {0} [built]
    + 25 hidden modules
Serverless: Watching with Webpack...
Serverless: scheduler: scheduling integrationJobCreator/integrationJobCreator with */1 * * * *
Serverless: scheduler: scheduling storyFanOutJobCreator/storyFanOutJobCreator with */1 * * * *
Serverless: Starting Offline: development/us-east-1.

Serverless: Routes for integrationJobCreator:
Serverless: (none)

Serverless: Routes for integrationJobWorker:
Serverless: (none)

Serverless: Routes for storyFanOutJobCreator:
Serverless: (none)

Serverless: Routes for storyFanOutJobWorker:
Serverless: (none)

Serverless: Offline listening on http://localhost:9000
Hash: 1a2c2e8f9237050c650b
Version: webpack 3.5.5
Time: 649ms
     Asset     Size  Chunks             Chunk Names
handler.js  70.9 kB       0  [emitted]  main
   [0] external "babel-runtime/regenerator" 42 bytes {0} [not cacheable]
   [2] external "redbird-core/lib/constants" 42 bytes {0} [not cacheable]
   [4] ./src/helpers/connectToDatabase.js 1.57 kB {0} [built]
   [5] external "async/queue" 42 bytes {0} [not cacheable]
   [6] external "babel-runtime/core-js/json/stringify" 42 bytes {0} [not cacheable]
   [7] external "aws-sdk" 42 bytes {0} [not cacheable]
   [8] ./src/lambdas/integrationJobWorker/index.js 7.18 kB {0} [built]
  [12] ./src/lambdas/storyFanOutJobWorker/index.js 6.96 kB {0} [built]
  [13] ./src/handler.js 1.17 kB {0} [built]
  [14] ./src/lambdas/integrationJobCreator/index.js 3.92 kB {0} [built]
  [33] ./src/lambdas/integrationJobWorker/updateProfileValue.js 2.83 kB {0} [built]
  [36] ./src/lambdas/integrationJobWorker/updateTickerValue.js 2.47 kB {0} [built]
  [37] ./src/lambdas/storyFanOutJobCreator/index.js 3.92 kB {0} [built]
  [38] ./src/lambdas/storyFanOutJobWorker/addToNewsFeed.js 3.05 kB {0} [built]
  [39] ./src/lambdas/storyFanOutJobWorker/updateFanOutStatus.js 2.45 kB {0} [built]
    + 25 hidden modules
Serverless: scheduler: unable to find source for storyFanOutJobCreator
Serverless: scheduler: unable to find source for integrationJobCreator
Serverless: scheduler: unable to find source for integrationJobCreator
Serverless: scheduler: unable to find source for storyFanOutJobCreator
Serverless: scheduler: unable to find source for storyFanOutJobCreator
Serverless: scheduler: unable to find source for integrationJobCreator
Serverless: scheduler: unable to find source for integrationJobCreator
Serverless: scheduler: unable to find source for storyFanOutJobCreator
Serverless: scheduler: unable to find source for storyFanOutJobCreator
Serverless: scheduler: unable to find source for integrationJobCreator
```

**Comment by joncursi** [comment]
Fixed via https://github.com/ajmath/serverless-offline-scheduler/issues/11#issuecomment-362802623

**Comment by ajmath** [comment]
Published in 0.3.5
